client/login/wp-login/footer.jsx

```diff
@@ -0,0 +1,70 @@
+/**
```

Comment:
I'm concerned about extracting this component for use elsewhere but leaving it in the `wp-login` directory.

Ideally, shared components should move to either [`blocks`](https://github.com/Automattic/wp-calypso/blob/master/client/blocks/README.md) or [`components`](https://github.com/Automattic/wp-calypso/blob/master/client/components/README.md). Since this is connected, it's probably a good fit for the `blocks` directory, although the distinction is a bit ambiguous and the case could be made that it should go in `components` because it's quite simple.

In that case, its relevant styles should accompany it. It may make sense to do all of this in an initial PR, then follow-up adding it to Jetpack connect, but it's up to you, I don't think it will get too big.

**Aside**: I find it nice to use `prettier`, you can enable automatic formatting and forget about lint errors by adding `/** @format */` as the first line in any file. I try to add it to new files which is why I mention it here. It's probably best to avoid it in existing files in PRs with other concerns because it will inflate the diff.

Reply:
I've now moved the component to `components`, and it uses the `JetpackColophon` component to display the Jetpack footer. 

It should now be easier to extend that footer in the future.